<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>672</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Playlist Browser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/play-button.png</normaloff>../resources/play-button.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="list_categ_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>256</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_categ_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>256</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_channels">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>256</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_watch">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Watch</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_groups">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Groups</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_categories">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>15</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Categories</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_channels">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>15</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Channels</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wait">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>LOADING...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_filter_groups">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_filter_categories">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>10</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_filter_channels">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>10</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_url">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>711</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frm_catchup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>821</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTimeEdit" name="time_start">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk_catchup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Catch up replay</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_start">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_catchup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>No catchup available</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb_epg">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>401</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_duration">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_api"/>
    <addaction name="action_Open_local_file"/>
    <addaction name="action_Open_remote_file"/>
    <addaction name="separator"/>
    <addaction name="action_Preferences"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_api"/>
   <addaction name="action_Open_local_file"/>
   <addaction name="action_Open_remote_file"/>
   <addaction name="separator"/>
   <addaction name="action_Preferences"/>
  </widget>
  <action name="action_api">
   <property name="icon">
    <iconset>
     <normaloff>../resources/login.png</normaloff>../resources/login.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Xstream Code API...</string>
   </property>
  </action>
  <action name="action_Open_local_file">
   <property name="icon">
    <iconset>
     <normaloff>../resources/folder.png</normaloff>../resources/folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open local file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Open_remote_file">
   <property name="icon">
    <iconset>
     <normaloff>../resources/link.png</normaloff>../resources/link.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;remote file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="icon">
    <iconset>
     <normaloff>../resources/gear.png</normaloff>../resources/gear.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
